app/views/payment-schemes.njk

```diff
@@ -23,7 +23,7 @@
         </div>
       {% endif %}
 
-      {% if model.head.length %}
+      {% if model.rows.length > 0 %}
```

Comment:
Do not show the table based on the content rather than the header items (which are always present).